opm/simulators/linalg/MILU.hpp

```diff
@@ -86,14 +106,14 @@ struct RealReorderer : public Reorderer
 
   
 template<class M>
-void milu0_decomposition(M& A, std::function<double(const double&)> absFunctor = AuxFunctors::signFunctor(),
-			  std::function<double(const double&)> signFunctor = AuxFunctors::oneFunctor(),
+void milu0_decomposition(M& A, double absFunctor(const double&) = signFunctor<Opm::detail<double>>(const double&),
+			 double signFunctor(const double&) = oneFunctor<Opm::detail<double>>(const double&),
                          std::vector<typename M::block_type>* diagonal = nullptr);
 
 template<class M>
 void  milu0_decomposition(M& A, std::vector<typename M::block_type>* diagonal)    
  {
-   milu0_decomposition(A, AuxFunctors::identityFunctor(), AuxFunctors::oneFunctor(), diagonal);
+   milu0_decomposition(A, identityFunctor<Opm::detail<double>>(const double&), oneFunctor<Opm::detail<double>>(const double&), diagonal);
```

Comment:
dito

Reply:
new version with std::functions is compiling, so far with no errors.... let's see 
PS: I also just leant what dito is :)

Reply:
that is of course elementary/vital knowledge, How you could have survived without it before is beyond me. :laughing: :rofl: